File: Practice/15/C++/15. Угадай число/15. Угадай число/15. Угадай число.cpp
#include <iostream>
#include <cstdlib>
#include<ctime>
int main()
{
    setlocale(LC_ALL, "russian");
    srand(time(NULL));
    int number = rand()%101;
    int choice;
    std::cout << "Привествую в игре!" << "Угадай число" << std::endl << "Начнем?\n\"ДА-1\"\n\"НЕТ-любое другое число\"" << std::endl;
    int pick;
    std::cin >> pick;
    again:
    int i=0;
    for (i;pick==1 ;i=i) {
        while (pick == 1 && i < 5)
        {
            std::cout << "Введите число: ";
            std::cin >> choice;
            std::cout << std::endl;

            if (choice < number)
            {
                std::cout << "Загаданное число больше" << std::endl;
            }
            else if (choice > number)
            {
                std::cout << "Загаданное число меньше" << std::endl;
            }
            else if (choice == number)
            {
                pick = 0;
                std::cout << "Поздравляю! Вы угадали\nХотите начать еще?\n\"ДА-1\"\"НЕТ-любое другое число\"" << std::endl;
                std::cin >> pick;
                if (pick == 1) {
                    continue ;
                }
                else {
                    break;
                }
            }
            i++;
        }
        if (i == 5)
        {
            std::cout << "Вы проиграли(( "<<"Загаданное число ="<<number<<"\nХотите начать еще?\n\"ДА-1\"\"НЕТ-любое другое число\"" << std::endl;
            std::cin >> pick;
            if (pick == 1) {
                goto again;
            }
            else {
                break;
            }
        }
    }
}

File: Practice/15/Python/15. Угадай число/15. Угадай число/_15._Угадай_число.py
from random import randint

try:
    a = randint(0, 100)
    print("Приветствую в игре\n\"Угадай число\"")
    pick=int(input("Начнем?\n\"ДА-1\"\n\"НЕТ-0\"\n"))
    i = 0
    while pick == 1:
        choice = int(input("Введи число: "))
        if choice < a:
            print("Загаданное число больше")
        elif choice > a:
            print("Загаданное число меньше")
        else:
            print("Поздравляю! Вы угадали")
            pick = int(input("Хотите попробовать еще раз?\nЕсли Да, введите - 1\nЕсли нет, введите - 0\n"))
            a = randint(0, 100)
            i = 0
        i += 1
        if i == 5:
            print("Вы проиграли,загаданное число:",a)
            pick = int(input("Хотите попробовать еще раз?\nЕсли Да, введите - 1\nЕсли нет, введите - 0\n"))
            a = randint(0, 100)
            i = 0
except ValueError:
    print ("Неверный ввод!")